tdp_vars_defaults/grafana/grafana.yml

```diff
@@ -49,6 +50,12 @@ grafana_datasources:
     basicAuthUser: "{{ prometheus_admin_user }}"
     secureJsonData:
       basicAuthPassword: "{{ prometheus_admin_password }}"
+  - name: TDP Loki
+    type: loki
+    default: yes
```

Comment:
Prometheus datasource (defined above) is already default. Is that not an issue ?